regparser/layer/terms.py

```diff
@@ -3,7 +3,12 @@
 from itertools import chain
 import re
 
-from inflection import pluralize
+import inflection 
+try:
+    del inflection.PLURALS[inflection.PLURALS.index(('(?i)(p)erson$', '\\1eople'))]
+except ValueError:
+    pass
```

Comment:
I'm a bit confused. Is this saying that we don't want `person` to be pluralized to `people`? We want to catch `peoples`? Is there a way to do this which would catch both `people` and `peoples`?